image_test/sbom/test_sbom_export.sh

```diff
@@ -0,0 +1,36 @@
+URL="http://metadata/computeMetadata/v1/instance/attributes"
```

Comment:
should this file start with shebang (`#!/bin/bash`) ?

Reply:
It might be better to provide some context:

The purpose of this file is so that the steps to generate the SBOM itself (running syft) are all in one location, even though Debian and Enterprise Linux have different mounting methodologies.

From debugging when running this script in #2036, the script needed the shebang when called from a python subprocess, or error 8 would result. For this script to be used in both Debian and Enterprise Linux, it would need to keep the shebang.

Reply:
Actually I was looking at the wrong file, I think it should have the shebang.